ui/packages/catalog-ui-search/src/main/webapp/component/map-info/map-info.view.js

```diff
@@ -72,66 +86,63 @@ module.exports = Marionette.LayoutView.extend({
   handleTarget: function() {
     this.$el.toggleClass('has-feature', this.model.get('target') !== undefined)
   },
-  serializeData: function() {
-    let modelJSON = this.model.toJSON()
-    let summaryModel = {}
-    const summary = properties.summaryShow
-
-    summary.forEach(attribute => {
-      if (this.model.get('targetMetacard') !== undefined) {
-        const attributeName = hbHelper.getAlias(attribute)
-        const attributeValue = this.model
-          .get('targetMetacard')
-          .get('metacard')
-          .get('properties')
-          .get(attributeName)
-        if (attributeValue !== undefined)
-          summaryModel[attributeName] = massageData(
-            attributeName,
-            attributeValue
-          )
-      }
-    })
-
-    let viewData = {
-      summary: summaryModel,
-      target: modelJSON.target,
-      lat: modelJSON.mouseLat,
-      lon: modelJSON.mouseLon,
+  getDisplayComponent: function() {
+    const coordinateFormat = getCoordinateFormat()
+    const lat = this.model.get('mouseLat')
+    const lon = this.model.get('mouseLon')
+    if (typeof lat === 'undefined' || typeof lon === 'undefined') {
+      return null
     }
-    switch (getCoordinateFormat()) {
+    switch (coordinateFormat) {
       case 'degrees':
-        viewData.lat =
-          typeof viewData.lat === 'undefined'
-            ? viewData.lat
-            : mtgeo.toLat(viewData.lat)
-        viewData.lon =
-          typeof viewData.lon === 'undefined'
-            ? viewData.lon
-            : mtgeo.toLon(viewData.lon)
-        break
+        return <span>{mtgeo.toLat(lat) + ' ' + mtgeo.toLon(lon)}</span>
       case 'decimal':
-        viewData.lat =
-          typeof viewData.lat === 'undefined'
-            ? viewData.lat
-            : `${leftPad(Math.floor(viewData.lat), 3, ' ')}.${Math.abs(
-                viewData.lat % 1
-              )
-                .toFixed(6)
-                .toString()
-                .slice(2)}`
-        viewData.lon =
-          typeof viewData.lon === 'undefined'
-            ? viewData.lon
-            : `${leftPad(Math.floor(viewData.lon), 4, ' ')}.${Math.abs(
-                viewData.lon % 1
-              )
-                .toFixed(6)
-                .toString()
-                .slice(2)}`
-        break
+        return this.decimalComponent(lat, lon)
+      case 'mgrs':
+        // TODO: Move leaking defensive check knowledge to usng library (DDF-4335)
+        return lat > 84 || lat < -80 ? (
+          'In UPS Space'
+        ) : (
+          <span>{converter.LLtoUSNG(lat, lon, usngPrecision)}</span>
+        )
+      case 'utm':
+        return <span>{converter.LLtoUTMUPS(lat, lon)}</span>
+    }
+    throw 'Unrecognized coordinate format value [' + coordinateFormat + ']'
```

Comment:
❓ 	Should we show something like this in the console but also default to one of the other formats?

Reply:
Good question. This is mostly meant to be a red flag for us as we iterate over these components. If this gets thrown it'll be hard to miss. If we default we might think that things are proceeding as normal and have no idea an invalid value is being passed.